test/xpu_api/iterators/predef.iterators/reverse.iterators/reverse.iter.ops/reverse.iter.op.ref/op_arrow_rev_iter_op_ref.pass.cpp

```diff
@@ -144,12 +139,12 @@ kernel_test()
 }
 
 int
-main(int, char**)
+main()
 {
+#ifdef TEST_DPCPP_BACKEND_PRESENT
     auto ret = kernel_test();
-    if (ret)
-        std::cout << "Pass" << std::endl;
-    else
-        std::cout << "Fail" << std::endl;
-    return 0;
+    EXPECT_TRUE(ret, "Wrong result of work in kernel_test()");
```

Comment:
it makes sense to add more details in the message.

Reply:
Fixed